[2024-08-07T15:52:21.871+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-07T15:52:21.892+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_prueba_tecnica.limpieza manual__2024-08-07T15:09:28.390897+00:00 [queued]>
[2024-08-07T15:52:21.898+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_prueba_tecnica.limpieza manual__2024-08-07T15:09:28.390897+00:00 [queued]>
[2024-08-07T15:52:21.898+0000] {taskinstance.py:2306} INFO - Starting attempt 7 of 8
[2024-08-07T15:52:21.910+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): limpieza> on 2024-08-07 15:09:28.390897+00:00
[2024-08-07T15:52:21.918+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=376) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-07T15:52:21.919+0000] {standard_task_runner.py:64} INFO - Started process 402 to run task
[2024-08-07T15:52:21.919+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_prueba_tecnica', 'limpieza', 'manual__2024-08-07T15:09:28.390897+00:00', '--job-id', '139', '--raw', '--subdir', 'DAGS_FOLDER/dag_prueba_tecnica.py', '--cfg-path', '/tmp/tmpfl0bunwt']
[2024-08-07T15:52:21.920+0000] {standard_task_runner.py:91} INFO - Job 139: Subtask limpieza
[2024-08-07T15:52:21.962+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_prueba_tecnica.limpieza manual__2024-08-07T15:09:28.390897+00:00 [running]> on host 278ba5711b2c
[2024-08-07T15:52:22.038+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='miguel_guimarey' AIRFLOW_CTX_DAG_ID='dag_prueba_tecnica' AIRFLOW_CTX_TASK_ID='limpieza' AIRFLOW_CTX_EXECUTION_DATE='2024-08-07T15:09:28.390897+00:00' AIRFLOW_CTX_TRY_NUMBER='7' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-07T15:09:28.390897+00:00'
[2024-08-07T15:52:22.039+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-07T15:52:49.233+0000] {warnings.py:112} WARNING - /opt/***/dags/dag_prueba_tecnica.py:37: FutureWarning: Passing literal json to 'read_json' is deprecated and will be removed in a future version. To read from a literal string, wrap it in a 'StringIO' object.
  df = pd.read_json(data, orient='records')

[2024-08-07T15:53:03.053+0000] {base.py:84} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-08-07T15:53:53.232+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-08-07T15:53:53.234+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-07T15:53:53.241+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_prueba_tecnica, task_id=limpieza, run_id=manual__2024-08-07T15:09:28.390897+00:00, execution_date=20240807T150928, start_date=20240807T155221, end_date=20240807T155353
[2024-08-07T15:53:53.395+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-07T15:53:53.410+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-07T15:53:53.414+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
